zones.go

```diff
@@ -16,23 +16,30 @@ type DownloadInfo struct {
 	Filename      string
 }
 
-// DownloadZone provided the zone download URL retrieved from GetLinks() downloads the zone file and
-// saves it to local disk at destinationPath
-func (c *Client) DownloadZone(url, destinationPath string) error {
+// DownloadZoneToWriter is analogus to DownloadZone but instead of writing it to a file, it will
+// write it to a provided io.Writer. It returns the number of bytes written to dest and any error
+// that was encountered.
+func (c *Client) DownloadZoneToWriter(url string, dest io.Writer) (int64, error) {
 	resp, err := c.apiRequest(true, "GET", url, nil)
 	if err != nil {
 		return err
```

Comment:
You need to return an int and error here.

Reply:
Oops, that's what I get for editing code from my phone. ;-) Sorry!